src/utils/__tests__/beforeBlockString.test.js

```diff
@@ -0,0 +1,48 @@
+import beforeBlockString from "../beforeBlockString"
+import postcss from "postcss"
+import sugarss from "sugarss"
+
+it("beforeBlockString rules", () => {
+  expect(postcssCheck("a {}")).toBe("a ")
+  expect(postcssCheck("\na\n{}")).toBe("\na\n")
+  expect(postcssCheck("\n\na,\nb,\n\tspan > .foo\n{}")).toBe("\n\na,\nb,\n\tspan > .foo\n")
+})
+
+it("beforeBlockString at-rules", () => {
+  expect(postcssCheck("@media print {}")).toBe("@media print ")
+  expect(postcssCheck("\n@media print, screen\n\t{}")).toBe("\n@media print, screen\n\t")
+  expect(postcssCheck("@supports (animation-name: test) {}")).toBe("@supports (animation-name: test) ")
+  expect(postcssCheck(
```

Comment:
I don't see an assertion here ... oh, now I see it. Could you put longer strings like this in variables so the `expect(..).toBe(..)` is more apparent?